api/tests/unit/application/preHandlers/models/CampaignAuthorization_test.js

```diff
@@ -36,4 +36,50 @@ describe('Unit | Domain | models | CampaignAuthorization', function () {
       expect(hasAccess).to.be.true;
     });
   });
+
+  describe('#isAllowedToAccess', function () {
+    it('should be true if user is member of organization and not owner of the campaign', function () {
```

Comment:
**suggestion** : 'should be true if user is member of organization and not owner of the campaign' => 'should be true if user is member of organization'